app.js

```diff
@@ -9,9 +9,7 @@ Test this function by hand in the console to get it working, and when you think
 function sum(a,b){ //eslint-disable-line
   var sumMath = a + b;
   var arrayString = ('The sum of ' + a + ' and ' + b + ' is ' + sumMath + '.');
-  console.log(arrayString);
   var arrayMath = [sumMath, arrayString];
```

Comment:
Instead of declaring this array as a variable, since it is only called in the return statement, simply returning the array e.g.
```
return [sumMath, arrayString]
```